Comanche055/TVCMASSPROP.agc

```diff
@@ -31,95 +31,100 @@
+#	VARST4 = INTVALUE4 + LEMMASS(SLOPEVAL4)		IAVG		SLOPE FOR CSMMASS > 33956 LBS ( SPS > 10000 LBS)
 #
-#		VARST3 = INTVALUE3 + LEMMASS(SLOPEVAL3)		IAVG/TLX	SLOPE FOR CSMMASS > 33956 LBS (SPS > 10000 LBS)
-#		VARST4 = INTVALUE4 + LEMMASS(SLOPEVAL4)		IAVG		SLOPE FOR CSMMASS > 33956 LBS (SPS > 10000 LBS)
+#	VARST5 = INTVALUE5 + LEMMASS(SLOPEVAL5)		IXX		SLOPE FOR ALL VALUES OF CSMMASS
 #
-#		VARST5 = INTVALUE5 + LEMMASS(SLOPEVAL5)		IXX		SLOPE FOR ALL VALUES OF CSMMASS
+#	VARST6 = INTVALUE6 + LEMMASS(SLOPEVAL6)		IAVG		SLOPE FOR CSMMASS < 33956 LBS ( SPS < 10000 LBS)
+#	VARST7 = INTVALUE7 + LEMMASS(SLOPEVAL7)		IAVG/TLX	SLOPE FOR CSMMASS < 33956 LBS ( SPS < 10000 LBS)
 #
-#		VARST6 = INTVALUE6 + LEMMASS(SLOPEVAL6)		IAVG		SLOPE FOR CSMMASS < 33956 LBS (SPS < 10000 LBS)
-#		VARST7 = INTVALUE7 + LEMMASS(SLOPEVAL7)		IAVG/TLX	SLOPE FOR CSMMASS < 33956 LBS (SPS < 10000 LBS)
+#	VARST8 = INTVALUE8 + LEMMASS(SLOPEVAL8)		IAVG		DECREMENT TO BRKPT VALUE WHEN LEM DSCNT STAGE OFF
+#	VARST9 = INTVALUE9 + LEMMASS(SLOPEVAL9)		IAVG/TLX	DECREMENT TO BRKPT VALUE WHEN LEM DSCNT STAGE OFF
 #
-#		VARST8 = INTVALUE8 + LEMMASS(SLOPEVAL8)		IAVG		DECREMENT TO BRKPT VALUE WHEN LEM DSCNT STAGE OFF
-#		VARST9 = INTVALUE9 + LEMMASS(SLOPEVAL9)		IAVG/TLX	DECREMENT TO BRKPT VALUE WHEN LEM DSCNT STAGE OFF
+#   (2) IF LEM NOT DOCKED
 #
-#	(2)	IF LEM NOT DOCKED
+#	VARST0 = NOLEMVAL0	WHERE THE MEANING AND SCALING OF VARST0
+#		.	.	TO VARST9 ARE THE SAME AS GIVEN ABOVE
+#		.	.
+#		.	.	NOTE... FOR THIS CASE, VARST8,9 HAVE NO
+#	VARST9 = NOLEMVAL9	MEANING (THEY ARE COMPUTED BUT NOT USED)
 #
-#		VARST0 = NOLEMVAL0	WHERE THE MEANING AND SCALING OF VARST0
-#			.		TO VARST9 ARE THE SAME AS GIVEN ABOVE
-#			.
-#			.		NOTE... FOR THIS CASE, VARST8,9 HAVE NO
-#		VARST9 = NOLEMVAL9	MEANING (THEY ARE COMPUTED BUT NOT USED)
+#   (3) THE FINAL OUTPUT CALCULATIONS ARE THEN DONE
 #
-# 	(3)	THE FINAL OUTPUT CALCULATIONS ARE THEN DONE
+#	IXX0 = VARST0 + (CSMMASS + NEGBPW)VARST5		IXX
 #
-#		IXX0 = VARST0 + (CSMMASS + NEGBPW)VARST5		IXX
+#	IXX1 = VARST1 + (CSMMASS + NEGBPW)VARST(4 OR 6)		IAVG
 #
-#		IXX1 = VARST1 + (CSMMASS + NEGBPW)VARST(4 OR 6)		IAVG
+#	IXX2 = VARST2 + (CSMMASS + NEGBPW)VARST(3 OR 7)		IAVG/TLX
 #
-#		IXX2 = VARST2 + (CSMMASS + NEGBPW)VARST(3 OR 7)		IAVG/TLX
 #
-# 	THE DATA USED CAME FROM THE CSM/LM SPACECRAFT OPERATIONAL DATA BOOK
-#		VOL. 3, NASA DOCUMENT SNA-8-D-027 (MARCH 1968)
+# THE DATA USED CAME FROM CSM/LM SPACECRAFT OPERATIONAL DATA BOOK.
+#	VOL. 3, NASA DOCUMENT SNA-8-D-027 (MARCH 1968)
 #
-# 	PERTINENT MASS DATA:		CSM WEIGHT	(FULL)	64100 LBS.
-#							(EMPTY)	23956 LBS.
-#					LEM WEIGHT	(FULL)	32000 LBS.
-#							(EMPTY)	14116 LBS.
+# PERTINENT MASS DATA :		CSM WEIGHT	(FULL)	64100 LBS.
+#						(EMPTY)	23956 LBS.
+#				LEM WEIGHT	(FULL)	32000 LBS.
+#						(EMPTY)	14116 LBS.
 #
-# 	(WEIGHTS ARE FROM AMMENDMENT #1 (APRIL 24, 1968) TO ABOVE DATA BOOK)
+# (WEIGHTS ARE FROM AMMNDMENT #1 (APRIL 24,1968) TO ABOVE DATA BOOK)
```

Comment:
⚠️ Change `AMMNDMENT` to `AMENDMENT` [[p952]](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0952.jpg)

Reply:
✅  Done.